### PR TITLE
Update index.tsx

### PR DESCRIPTION
**Pull request by jemsusem**
Making change to the Homepage TITLE tag.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const Home: NextPage = () => {
     <>
       <Head>
         <title>
-          Maryland Tree Trimming & Removal Services | Kuhns Tree Service{" "}
+          Tree Trimming & Removal Services in Silver Springs, MD | Kuhns Tree Service{" "}
```
Comment (jemsusem) [inline pages/index.tsx:26]: Total typo. I spelled Silver Spring wrong.

Comment (jlf0dev) [reply pages/index.tsx:26]: I think it's supposed to be `Silver Spring`. We've been having the same problem remembering 😄

Comment (jlf0dev) [reply pages/index.tsx:26]:
```suggestion
          Tree Trimming & Removal Services in Silver Spring, MD | Kuhns Tree Service{" "}
```

**Review by jlf0dev — CHANGES_REQUESTED** [review]

**Comment by jlf0dev** [comment]
Hi Alexander, I was doing some additional work to the website so I went ahead and changed the title on the homepage while I was at it. I'm going to close this PR since it won't apply, but feel free to open it again if I missed anything!

I should also mention that I switched us over to use a npm package called `next-seo` in #15. This handles a lot of the boilerplate we would have to write for each page, and instead consolidates it all into one place. I recommend checking out the PR (and possibly the [docs](https://github.com/garmeeh/next-seo)) since you'll likely be messing with those tags in the future.

**Pull request closed without merging**